### PR TITLE
Feat/add new datastructures

### PR DESCRIPTION
**Pull request by fakemonster**

### CODE REVIEW

File: src/models/debt/defaultDebt.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import defaultInterestRate from '../interest_rate/defaultInterestRate'
+
+const defaultDebt = () => ({
+  title: '',
+  amountOwed: 0,
+  interestRate: defaultInterestRate()})
```
Comment (clarkenciel) [inline src/models/debt/defaultDebt.js]: can we put the `})` on a new line here?

**Review by clarkenciel — APPROVED** [review]
this looks really good!

**Comment by clarkenciel** [comment]
oh also, don't forget to squash your commits. We probably only need the first commit message.

File: src/models/debt/defaultDebt.js (hunk 1 above)
Comment (fakemonster) [reply src/models/debt/defaultDebt.js]: yes, not sure how that got over there

**Comment by fakemonster** [comment]
lord almighty if i can figure out how to squash these i will

**Pull request merged**